samples/rust/pings.yaml

```diff
@@ -20,3 +20,4 @@ prototype:
     - N/A
   notification_emails:
     - CHANGE-ME@example.com
+  enabled: true
```

Comment:
If it's optional, we can omit.

Reply:
Yep, this was an artifact from some of my testing that I overlooked when cleaning things up.